src/sys/context.rs

```diff
@@ -0,0 +1,25 @@
+use libc;
+use {Errno, Result};
+use std::mem;
+
+#[derive(Clone, Copy)]
+pub struct UContext {
+    context: libc::ucontext_t,
```

Comment:
for direct wrappers like this, I kind of prefer not naming the inner field, and just accessing it via `.0`. thoughts?